app/controllers/events/view/tickets/add-order.js

```diff
@@ -58,6 +58,9 @@ export default Controller.extend({
       let order = this.get('model.order');
       let event = order.get('event');
       order.tickets.forEach(ticket => {
+        if (ticket.type === 'donation' && ticket.price === 0) {
+          this.notify.error(this.l10n.t('Price of a donation ticket needs to be greater than 0'));
```

Comment:
no, don't randomly insert an error message here, it's not blocking anything either way.